src/views/explore/explore.jsx

```diff
@@ -0,0 +1,159 @@
+var injectIntl = require('react-intl').injectIntl;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var React = require('react');
+var render = require('../../lib/render.jsx');
+
+var Api = require('../../mixins/api.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+var Box = require('../../components/box/box.jsx');
+var Tabs = require('../../components/tabs/tabs.jsx');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+var Grid = require('../../components/grid/grid.jsx');
+
+require('./explore.scss');
+
+// @todo migrate to React-Router once available
```

Comment:
@rschamp @thisandagain I think we should chat briefly about this at some point. When I looked at it last week it seemed pretty straigthforward to do a multi-page app using router and webpack together w/ chunking the bundle, but I bet @rschamp might have some more insight after the webpack work you've done recently!